perf/benchmarkapps/NativeServer/Program.cs

```diff
@@ -38,10 +33,16 @@ public static void Main(string[] args)
                .AddCommandLine(args)
                .Build();
 
+            var protocol = config["protocol"] ?? string.Empty;
+            if (!protocol.Equals("h2c", StringComparison.OrdinalIgnoreCase))
+            {
+                throw new InvalidOperationException("Only h2c is supported by C-core benchmark server.");
```

Comment:
Is this just how we're setting up the benchmarks server? Or is there another limitation that's causing this?

Reply:
Getting TLS certificates to work in Grpc.Core is a pain in the butt so I just didn't do it. Can change in the future.